osu.Game/Beatmaps/BeatmapSetOnlineInfo.cs

```diff
@@ -72,16 +73,67 @@ public class BeatmapSetOnlineInfo
         public BeatmapSetOnlineAvailability Availability { get; set; }
 
         /// <summary>
-        /// Beatmap set's song genre.
+        /// The song genre of this beatmap set.
         /// </summary>
         public BeatmapSetOnlineGenre Genre { get; set; }
 
         /// <summary>
-        /// Beatmap set's song language.
+        /// The song language of this beatmap set.
         /// </summary>
         public BeatmapSetOnlineLanguage Language { get; set; }
     }
 
+    public enum BeatmapSetOnlineGenre
+    {
+        [Description("Any")]
```

Comment:
i don't think you need to assign a description to enums that don't need, `GetDescription` to an enum that doesn't have description will return the name instead

Reply:
https://github.com/ppy/osu/blob/master/osu.Game/Users/User.cs#L190

Reply:
After a discussion about that, you can keep it as it'll be safer if someone changed the internal name.